test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -48,6 +48,8 @@
     CPhaseGate,
 )
 from qiskit.transpiler.passes import ApplyLayout
+from qiskit.circuit.classicalfunction import classical_function
+from qiskit.circuit.classicalfunction.types import Int1
```

Comment:
Tweedledum isn't a required import, so these lines need to be protected within the function, and it needs decorating with a `@unittest.skipUnless(optionals.HAS_TWEEDLEDUM, ...)` line.

Reply:
oh! I forgot. Fixed in 91069adaf8d686b750bb58f5ff4069fa74d1d52f